devel/kcov/Portfile

```diff
@@ -41,3 +41,9 @@ patchfiles          bin-to-c-source.py.use-python38.patch
 post-patch {
     reinplace "s|@PREFIX@|${prefix}|g" ${worksrcpath}/src/bin-to-c-source.py
 }
+
+post-destroot {
+    # kcov links to LLDB as @rpath/LLDB.framework/Versions/A/LLDB but doesn't add an rpath to find it
+    # On at least OS X 11 and 12 this causes kcov to fail to start with a dynamic linker error
+    system "install_name_tool -add_rpath /Library/Developer/CommandLineTools/Library/PrivateFrameworks ${destroot}${prefix}/bin/kcov"
```

Comment:
Seems reasonable, at least on the surface. But unfortunately I don't have a Big Sur or Monterey install to test against.

Also, is there a preferred/better way to handle this type of situation? @kencu and others?